src/pages/guides/using/asset-events/asset-events-properties.md

```diff
@@ -54,7 +54,7 @@ An object containing all the Resource Change objects related to the event. Resou
 
 **&lt;link relation&gt;** _object_
 
-The Resource Change object, which describes how a particular [resource](asset-events-glossary.md#resource) was affected by the [action](asset-event-actions.md) that triggered the event.
+The Resource Change object, which describes how a particular [resource](asset-events-glossary.md#resource) was affected by the [action](/src/pages/guides/using/asset-events/asset-events-actions.md) that triggered the event.
```

Comment:
Ah not really. The thing is that both absolute and relative links work. When I was fixing the broken links, I preferrred using absolute urls instead of relative ones, because they were starting to get very confused varying `../` notations.
Here, I could actually get rid of `/src/pages/guides/using/asset-events/asset-events/` prefix. WDYT?

Reply:
Yes. Since the file is in the same directory, lets consistently use relative links. 👍

Reply:
Done.